content/get-haiku/r1beta4/release-notes.md

```diff
@@ -92,6 +92,10 @@ The new diver is much more idiomatic, supports the major layers of Haiku's commo
 
 There is now an AVIF image translator available by default, providing both read and write support for this new and advanced image format for all native Haiku applications.
 
+### WebPositive
+
+HaikuWebKit has been updated to a much newer version of upstream WebKit, and contains many bugfixes since the previous release of Haiku. It also now uses the cURL networking backend, which resolved a large number of the network-related issues seen in previous releases.
```

Comment:
we can say "up to date" instead of "much newer". It is probably actually possibly even newer than what Apple ships in Safari, being just a few weeks old. Otherwise, it seems OK. I would like to be more ambitious in the claims here but I'm not sure we can :)